deployment/config.go

```diff
@@ -0,0 +1,19 @@
+package deployment
+
+import "github.com/astronomer/astro-cli/houston"
+
+var (
+	appConfig    *houston.AppConfig
+	appConfigErr error
+)
+
+var GetAppConfig = func(client houston.ClientInterface) (*houston.AppConfig, error) {
```

Comment:
I was thinking about doing something like that but I'm not sure the "Storage/caching" mechanism should be a part of this.

What if we want to reuse this houston.GetAppConfig later on to force a new call to get the app config ? 

Since this is not the case yet, I don't want to over-engineer the whole project, so I'll just change the code as you suggested. If in the futur we need to do a new call to Houston API without cache, we'll just add a bool param to the function.